package.json

```diff
@@ -225,12 +224,12 @@
 		"test-unit-php-multisite": "cross-env WP_MULTISITE=1 wp-scripts env test-php",
 		"test-unit:native": "cd test/native/ && cross-env NODE_ENV=test jest --config ./jest.config.js",
 		"test-unit:native:debug": "cd test/native/ && node --inspect ../../node_modules/.bin/jest --runInBand --config ./jest.config.js",
+		"design-system:build": "npm run build:packages && build-storybook -c ./packages/components/storybook -o ./playground/dist/design-system/components",
```

Comment:
This means we'll build packages twice when deploying, can we somehow only do it once?

Reply:
I guess we should consolidate the playground in Storybook as a next step. There is no reason to keep it as a standalone app.

Reply:
That makes sense.